dsl/es-lang-c-stdc99.c

```diff
@@ -688,7 +688,7 @@ es_symbol_p    (const EsObject*   object)
   return es_object_type_p(object, ES_TYPE_SYMBOL);
 }
 
-const char*
+static const char*
 es_symbol_name  (const EsObject*   object)
 {
```

Comment:
Could you keep this "extern" ?
I found this function should be named es_symbol_get. es_symbol_get is declared in es-lang-c-stdc99.h.

Reply:
Fixed, and renamed the function in another commit